smrf/distribute/solar.py

```diff
@@ -243,6 +248,10 @@ def initialize(self, topo, data):
         self.veg_tau = topo.veg_tau
         self.veg_k = topo.veg_k
 
+        # self.thread_variables = self.CLEAR_SKY_THREAD_VARIABLES + \
+        #     self.CLOUD_THREAD_VARIABLES + self.VEG_THREAD_VARIABLES + \
+        #     ['net_solar']
+
```

Comment:
Leftover from testing?

Reply:
This was the original location for it before I moved it to a class constant.